biogrid/libs/components/src/lib/components.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './components.css';
+
+/* eslint-disable-next-line */
```

Comment:
What rule are you disabling here?

Reply:
Auto added, will remove

Reply:
Oh the linter flags empty interfaces, I added a property